README.md

```diff
@@ -1,83 +1,40 @@
 [![Build Status](https://travis-ci.org/DataDog/dd-agent.svg?branch=master)](https://travis-ci.org/DataDog/dd-agent)
 
-The Datadog Agent faithfully collects events and metrics and brings
-them to [Datadog](https://app.datadoghq.com) on your behalf so that
-you can do something useful with your monitoring and performance data.
+# Important note
 
-You're looking at the source code right now. We provide a number of
-pre-packaged binaries for your convenience for both [Agent 5](https://app.datadoghq.com/account/settings?agent_version=5#agent) and [Agent 6]((https://app.datadoghq.com/account/settings#agent)). 
+This repository contains the source code for the Datadog Agent until major version 5.
```

Comment:
`up to and including` sounds better to me than `until`, but don't know why exactly 🤔

Reply:
because it's less ambiguous? Indeed it's not clear if `until` has to be considered `<=` or `<`

Reply:
yup exactly